CONTRIBUTING.md

```diff
@@ -17,6 +17,45 @@ These are, in general, rules that you should be following while contributing to
 - Issues would be assigned on a "First Come, First Served" basis.
 - Do mention (@maintainer) the project maintainer if your PR isn't reviewed within a few days.
 
+## Build Instructions
+
+- Fork & clone the project (please see below for instructions on how to do this).
```

Comment:
where are the instructions ? We can remove this most likely 

> (please see below for instructions on how to do this).

Reply:
I meant the intructions under "First time contributors" section. alright, I'll remove.